Formula/odpi.rb

```diff
@@ -3,6 +3,8 @@ class Odpi < Formula
   homepage "https://oracle.github.io/odpi/"
   url "https://github.com/oracle/odpi/archive/v4.0.1.tar.gz"
   sha256 "7e4b9f09607e80f9836990678cd45a542eaedf99c3334e1c6f8956ffc5d88c02"
+  # license ["Apache-2.0", "UPL-1.0"] - pending https://github.com/Homebrew/brew/pull/7953
+  license "Apache-2.0"
```

Comment:
Just a reminder that multiple licenses are allowed now.

This won't block this PR, though, as I know you're working on a PR for these.

Reply:
brew 2.4.10 will need to release first.

Reply:
Yes, good catch